README.md

```diff
@@ -8,6 +8,7 @@
 |  <= 0.0.6   |  0.191.1 |
 |  0.0.7      |  0.194.1 |
 |  0.0.8 <=   |  0.195.0 |
```

Comment:
```suggestion
|  >= 0.0.9  |  0.195.0 |
```
This field should be auto generated somehow based on the `gradle.properties` file. I need to look into it in the future.